src/views/Completed.vue

```diff
@@ -576,6 +579,21 @@ export default defineComponent({
       }
 
       order.isGeneratingShippingLabel = false;
+    },
+    async showShippingLabelErrorModal(order: any){
+      // Getting all the shipment ids and fetching the data of shipping label error by passing them
+      const shipmentIds = order.shipments.map((shipment: any) => shipment.shipmentId);
```

Comment:
We can get the order shipment ids directly from order.shipmentIds

Reply:
We can move the logic to fetch the shipment label error messages to the modal component as we are not using the messages anywhere in this component.